### PR TITLE
core-1403 Allow users to copy DE and public link from Details drawer.

### PR DESCRIPTION
**Pull request by sri2k1us**
image

### CODE REVIEW

File: src/components/data/PublicLinks.js [hunk 1]
```diff
@@ -161,7 +161,7 @@ function PublicLinks(props) {
             <Paper className={classes.paper} elevation={0}>
                 <TextField
                     multiline
-                    rows={paths?.length || 3}
+                    rows={paths?.length + 1 || 3}
```
Comment (psarando) [inline src/components/data/PublicLinks.js:164]: 👍

**Review by psarando — APPROVED** [review]
LGTM :shipit: 👍

File: src/components/data/details/DetailsPanel.js [hunk 2]
```diff
@@ -178,11 +171,38 @@ function DetailsTabPanel(props) {
...
+                <GridLabelValue label={t("deLink")}>
+                    <CopyTextArea
+                        text={`${getHost()}${partialLink}`}
+                        debugIdPrefix={build(baseId, ids.md5)}
+                    />
+                </GridLabelValue>
+                {isFile && (
+                    <Grid item xs={6}>
+                        <Button
+                            color="primary"
```
Comment (ash3rz) [inline src/components/data/details/DetailsPanel.js:197]: Minor, but always nice to have an ID for testing

File: src/components/utils/GridLabelValue.js [hunk 3]
```diff
@@ -10,10 +10,10 @@ import { Grid, Typography } from "@material-ui/core";
 
 // A grid where the secondary column, values, intentionally take up more
 // space than the first column, labels
-export default function GridLabelValue({ label, children, ...props }) {
+export default function GridLabelValue({ id, label, children, ...props }) {
```
Comment (ash3rz) [inline src/components/utils/GridLabelValue.js:13]: 👍

**Review by ash3rz — APPROVED** [review]
:shipit:

**Comment by sri2k1us** [comment]
Thanks for the reviews.

**Pull request merged**